meta-dts-distro/recipes-extended/hw-probe/hw-probe_1.6.5.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "tool to probe for hardware, check operability and find drivers"
+HOMEPAGE = "https://github.com/linuxhw/hw-probe.git"
+LICENSE = "LGPL-2.1-or-later"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/LGPL-2.1-or-later;md5=2a4f4fd2128ea2f65047ee63fbca9f68"
+
+SRC_URI = "https://github.com/linuxhw/hw-probe/releases/download/1.6/hw-probe-1.6.5-189-x86_64.AppImage;sha256sum=dddec100c47b642c4bdd4f578b05bc8890c4f0ecbd0d5f15e965927590fad505"
```

Comment:
@TomaszAIR well the installation process for `hw-probe` is just copying the perl script to `/usr/bin`.
and from the [install from source](https://github.com/linuxhw/hw-probe/blob/master/INSTALL.md#install-from-source) section we can see that there are many required dependencies for it to work.

My thought is that instead of installing all of those, we can just grab the AppImage.

Reply:
Let's stick with AppImage for a start and see how it works in practice.